osquery/tables/system/linux/shadow.cpp

```diff
@@ -40,6 +40,8 @@ void genShadowForAccount(const struct spwd* spwd, QueryData& results) {
       r["password_status"] = "not_set";
     } else if (password[0] == '!' || password[0] == '*' || password[0] == 'x') {
       r["password_status"] = "locked";
+    } else if (password == "") {
```

Comment:
nitpick, can you use `password.empty()`?

Reply:
Sure! The magic of github suggests means I don't even have to check out the code again =)